airflow/api_connexion/endpoints/xcom_endpoint.py

```diff
@@ -26,18 +32,35 @@ def delete_xcom_entry():
     raise NotImplementedError("Not implemented yet.")
 
 
-def get_xcom_entries():
+@provide_session
+def get_xcom_entries(dag_id, dag_run_id, task_id, session):
```

Comment:
can you add type annotation to all handler functions?

Reply:
Hi @houqp , Is it necessary to add type annotation since connexion will be using it internally?

Reply:
@ephraimbuddy yes, type annotation helps catching bugs at build time for internal and external code.

Reply:
+1